[Doc Improvement][Get started update] (#8666)
MicrosoftDocs/msteams-docs: msteams-platform/get-started/choose-what-suits-you.md

A Teams app offers you the flexibility of choosing your build environment. It includes tools, framework, and languages to approach your app development. Build your Teams app in the environment that's best suited for your app. You can even select a combination. For example, you can use Teams Toolkit to build an app with JavaScript and host it on a SharePoint site.

> This section takes you through the various options that you've for building your Teams app as per your requirements.

Reviewer: Using "you've" before a preposition is not idiomatic English. This is a case where we should stay with "you have" or else put a verb like "got" between "you've" and the preposition "for".  Also, the "as per your requirements" is pointless.
```suggestion
This section takes you through the various options that you have for building your Teams app.
```
